src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -6562,4 +6690,54 @@ definitions:
       properties:
         type: object
         additionalProperties:
-            type: string
\ No newline at end of file
+            type: string
+  DownloadBodyCreate:
+    type: object
+    required:
+      - nodeIds
+    properties:
+      nodeIds:
+        type: array
+        items:
+          type: string
+          default: "278187d5-bf87-44ce-ab9a-3b4e6db932d1"
+  DownloadEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Download'
+  Download:
+    type: object
+    properties:
+      filesAdded:
+        type: integer
+        default: 0
+        description: number of files added so far in the zip
+      done:
```

Comment:
"done" is not very descriptive for someone just looking at the response JSON, I would suggest "bytesAdded".